.gitlab/lassen-jobs.yml

```diff
@@ -0,0 +1,70 @@
+###############################################################################
+# Copyright 2021 Lawrence Livermore National Security, LLC and other CARE
+# developers.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Based on implementation in the Umpire Project,  Copyright (c) 2018-20, 
+# Lawrence Livermore National Security, LLC and Umpire project contributors.
+#
+# See the COPYRIGHT file for details.
+###############################################################################
+
+# TODO: what variants do we want to actually test?
+
+# CPU ONLY
+ibm_clang_9:
+  variables:
+    SPEC: " %clang@9.0.0ibm"
+  extends: .build_and_test_on_lassen
+
+ibm_clang_9_gcc_8:
+  variables:
+    SPEC: " %clang@9.0.0ibm+tests cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1"
+  extends: .build_and_test_on_lassen
+
+gcc_8_3_1:
+  variables:
+    SPEC: "%gcc@8.3.1+tests"
+  extends: .build_and_test_on_lassen
+
+
+
+# CUDA - TODO - fix cuda tests and add + tests back to the tests here
+
+ibm_clang_9_cuda:
+  variables:
+#    SPEC: "+cuda +allow-unsupported-compilers cuda_arch=70 +tests %clang@9.0.0ibm^cuda@10.1.168 ^cmake@3.14.5"
+    SPEC: "+cuda +allow-unsupported-compilers cuda_arch=70 %clang@9.0.0ibm^cuda@10.1.168 ^cmake@3.14.5"
```

Comment:
Is cuda@10.1.243 supported?

Reply:
yeah, it should be. Do you want to switch to it?

Reply:
That's what our main host application uses...

Reply:
This will be addressed in a follow on pull request.